archetypes/data-playground.md

```diff
@@ -8,11 +8,11 @@ summary: "Add Summary To Display On List Page Here"
 github_link: "https://github.com/rotationalio/[insert-link-here]"
 description: "Add About This Data Source Description Here"
 producer_name: "Add Producer Name Here"
-producer_link: "https://finnhub.io"
+producer_link: "https://producer-website.com"
 data: "Add Details About the Data Provided Here"
 is_account_required: false
 license: "Add Data Source License Type Here (Ex. Free, Commercial)"
-license_warning: "Add License description if applicable"
+license_warning: Please review terms and conditions. Access to data sources can change.
```

Comment:
This is our disclaimer informing users that anything could change, so now this should be included any time a new source is added.